docs-source/content/userguide/introduction/introduction.md

```diff
@@ -23,18 +23,49 @@ Detailed instructions are available [here]({{< relref "/userguide/managing-opera
 * Oracle WebLogic Server 12.2.1.3.0 with patch 29135930.
    * The existing WebLogic Docker image, `container-registry.oracle.com/middleware/weblogic:12.2.1.3 `,
```

Comment:
there is an extra space after 12.2.1.3 and comma ","

Reply:
I think that it appears ok in the docs. The back tick confers code font and then the comma follows.
https://oracle.github.io/weblogic-kubernetes-operator/userguide/introduction/introduction/